<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Inserir_um_t√≠tulo-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCliente">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 230, 255);</string>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnimal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(136, 169, 39);</string>
    </property>
    <property name="text">
     <string>Animal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblfundo_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>971</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lblfundo_2</zorder>
   <zorder>lblLogo</zorder>
   <zorder>btnCliente</zorder>
   <zorder>btnAnimal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
